action.yml

```diff
@@ -89,6 +89,9 @@ inputs:
   diagnostics_retention_days:
     description: 'Number of days to keep the diagnostics files downloadable'
     required: false
+  network_air_gap:
+   description: 'Enable air gap'
```

Comment:
Network Air Gap to run synopsys-action on your network
something like that u can add, let others review as well